src/Microsoft.TemplateEngine.TemplateLocalizer.Core/Exceptions/JsonMemberMissingException.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace Microsoft.TemplateEngine.TemplateLocalizer.Core.Exceptions
+{
+    /// <summary>
+    /// Represents an exception thrown to indicate that a json element with the given name
+    /// was not found, even though it was expected to exist under the given parent json element.
+    /// </summary>
+    internal class JsonMemberMissingException : Exception
```

Comment:
minor: could this be sealed?
Resolve if the design expects descendants.
Similar comment for other unsealed classes